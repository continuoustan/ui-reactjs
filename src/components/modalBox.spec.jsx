import React from 'react';
import renderer from 'react-test-renderer';

import ModalBox from './modalBox';


test('Modal Box rendering', () => {
    const description = 'Modal Body description';
    const tree = renderer.create(
        <ModalBox description={description}>
            <button level={0} />
            <button level={1} />
            <button level={2} />
            <button level={3} />
        </ModalBox>
    ).toJSON();

    expect(tree).toMatchSnapshot();
});


