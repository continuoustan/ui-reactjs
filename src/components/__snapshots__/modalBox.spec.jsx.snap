// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Modal Box rendering 1`] = `
<div
  className="modalbox"
>
  <p
    className="modalbox--description"
  >
    Modal Body description
  </p>
  <div>
    <div
      className="modalbox--footer"
    >
      <button
        level={0}
      />
    </div>
    <div
      className="modalbox--footer"
    >
      <button
        level={1}
      />
    </div>
    <div
      className="modalbox--footer"
    >
      <button
        level={2}
      />
    </div>
    <div
      className="modalbox--footer"
    >
      <button
        level={3}
      />
    </div>
  </div>
</div>
`;
